src/components/AmountFilter.jsx

```diff
@@ -1,17 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { InputLabel, Grid } from '@mui/material';
+import { InputLabel, Grid, Slider } from '@mui/material';
 import { Remove as RemoveIcon } from '@mui/icons-material';
 import AmountTextField from './AmountTextField';
 
 export default function AmountFilter(props) {
-  const { min, max, onMinChange, onMaxChange } = props;
+  const { min, max, onMinChange, onMaxChange, onSliderChange } = props;
```

Comment:
Why not make onSliderChange a function of onMinChange and onMaxChange? The parent component doesn't need to know how min and max filters are implemented underneath. The parent only needs to be notified of changes in min or max